convert/src/tests.rs

```diff
@@ -0,0 +1,55 @@
+use super::*;
+
+#[test]
+fn convert_to_mongo_timestamp_and_back_test() {
+    let now = std::time::UNIX_EPOCH
+        + std::time::Duration::from_secs(
+            std::time::SystemTime::now()
+                .duration_since(std::time::UNIX_EPOCH)
+                .unwrap()
+                .as_secs(),
+        ); //This is needed because we need to discard anything below the second
+    let mongo_now = mongo::systime_to_timestamp(now);
+    assert_eq!(now, mongo::timestamp_to_systime(mongo_now));
+}
+#[test]
+fn convert_to_protobuf_timestamp_and_back_test() {
+    let now = std::time::SystemTime::now();
+    let proto_now = proto::systime_to_timestamp(now);
+    assert_eq!(now, proto::timestamp_to_systime(proto_now));
+}
```

Comment:
These tests are not really needed since the functions have already been unit tested below, but leave them if you wish

Reply:
They'll break in 2106 lol